frontend/app/components/ui/add-product-modal.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { Button } from "./button";
+import { Input } from "./input";
+import { Label } from "./label";
+import { Select, SelectTrigger, SelectValue, SelectContent, SelectItem } from "./select";
+import { Textarea } from "./textarea";
+
+interface ProductUploadModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const ProductUploadModal: React.FC<ProductUploadModalProps> = ({
```

Comment:
Rename the component to AddProductModal and here `React.FC<ProductUploadModalProps>` change it to   `React.FC<AddProductModalProps>`